### PR TITLE
COMRS-194 Generate CSV exports for FICHE DU PATIENT AVEC VIH - ADULTE in Initializer module formats

### PR DESCRIPTION
**Pull request by lauravignoli**
https://jembiprojects.jira.com/browse/COMRS-194

### CODE REVIEW

File: configuration/concepts/adult_hiv_initial_form.csv [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+6221e521-79c9-4a28-b3e5-e96df371b855,,,Age du dernier enfant,Age of last child,,,Misc,Numeric,,
+b2031349-b60a-433d-880a-8668cc040d2f,,,Hétérosexuel,Heterosexual,,,Misc,N/A,,
+9801e8cd-da10-4c52-adee-27cd6939adbb,,,Bisexuel,Bisexual,,,Misc,N/A,,
+c540d003-cdb5-434c-baa4-0b2b3e5e3205,,,Homosexuel,Homosexual,,,Misc,N/A,,
+211f0857-61a3-4049-9777-374c4a592453,,b2031349-b60a-433d-880a-8668cc040d2f;9801e8cd-da10-4c52-adee-27cd6939adbb;c540d003-cdb5-434c-baa4-0b2b3e5e3205,Orientation sexuelle,Sexual orientation ,,,Misc,Coded,,
+b7648479-e480-4772-b646-a5abc07bc4f7,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Partenaire informé,Informed partner,,,Misc,Coded,,
+7acfafa4-f19b-485e-97a7-c9e002dbe37a,,,Positif,Positive,,,Misc,N/A,,
+718b4589-2a11-4355-b8dc-aa668a93e098,,,Négatif,Negative,,,Misc,N/A,,
+b09900a6-4646-4804-b460-63be72879a13,,,Ne sait pas,Do not know,,,Misc,N/A,,
+351d2ae2-597e-471d-859c-ac0f782e06ec,,Positive;Negative;Do Not Know,Statut Partenaire,Status Partner,,,Misc,Coded,,
```
Comment (tmvumbi2) [inline configuration/concepts/adult_hiv_initial_form.csv:12]: @lauravignoli Could you please double check the above concept? Noticed the following issues:
`Fully specified name:fr` = `Positive;Negative` (the text is in English)
The corresponding English translation (`Do Not Know`) does not look right.
And also the short name seems to be unrelated to the fully specified name.

File: configuration/concepts/adult_hiv_initial_form.csv [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+b9bcbad0-3d5d-43f8-9cb8-b0a80ea440c7,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Partage des résultats sérologiques,Sharing of serological results,,,Misc,Coded,,
+3b75cc72-924b-4d36-8891-95484287495e,,,Inconu,Unknown,,,Misc,N/A,,
+9d26a59c-fe56-4509-b931-e380303a72c3,,,Non renseigné,Not specified,,,Misc,N/A,,
+2d1bf419-16a9-44a7-a11c-98d39b170cec,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e;Unknown;9d26a59c-fe56-4509-b931-e380303a72c3,Couples discordants,Discordant couples ,,,Misc,Coded,,
+d2b400ea-7510-432c-9ea4-27ea88736bb7,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e; ,Utilisation systématique du préservatif,Condom use systematically,,,Misc,Coded,,
```
Comment (tmvumbi2) [inline configuration/concepts/adult_hiv_initial_form.csv]: @lauravignoli Could you please remove the trailing ";" at the end of `b497171e-0410-4d8d-bbd4-7e1a8f8b504e; ,`

File: configuration/concepts/adult_hiv_initial_form.csv [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+faa8bd18-23b0-4988-9452-1a60094f50fa,,,Préciser,If other specify,,,Misc,Text,,
+597f78bc-09ac-4473-b686-7d81b26cf64d,,,VIH1,HIV1,,,Misc,N/A,,
+92cea782-a50b-439c-a3ae-1acc785c2c42,,,VIH2,HIV2,,,Misc,N/A,,
+de0efbb3-4f8c-4054-b834-23c2f8b75112,,,"VIH 1+2","HIV 1+2",,,Misc,N/A,,
+ca6e7017-241f-4d5e-ba17-efd280c1b6cd,,,VIH10,HIV10,,,Misc,N/A,,
+4cd723ee-83d1-4db8-86ca-eab4e7d220b0,,,VIH1N,HIV1N,,,Misc,N/A,,
+76cab961-e2ad-4d2f-bb00-03c73ca7ce11,,,VIH1P,HIV1P,,,Misc,N/A,,
+965b14e1-fbdb-4590-9e24-9fa5a64c68ce,,597f78bc-09ac-4473-b686-7d81b26cf64d;92cea782-a50b-439c-a3ae-1acc785c2c42;de0efbb3-4f8c-4054-b834-23c2f8b75112;ca6e7017-241f-4d5e-ba17-efd280c1b6cd;4cd723ee-83d1-4db8-86ca-eab4e7d220b0;76cab961-e2ad-4d2f-bb00-03c73ca7ce11,Type VIH,Type HIV,,,Misc,Coded,,
+23d94813-c123-4f6d-a7f8-66e22bc30e75,0d9692be-a4ea-48b8-bf44-3c6c8eb7ecf5;dcc5a64e-bf9f-402e-8ce3-42ccd3fb83ce;faa8bd18-23b0-4988-9452-1a60094f50fa;965b14e1-fbdb-4590-9e24-9fa5a64c68ce,,III.1 Test VIH,III.1 HIV TEST,,,Misc,N/A,,
+c6faef09-0a6c-4f3b-98c6-a47c7299cf5b,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e; ,Maladies opportunistes,Opportunitis disease,,,Misc,Coded,,
```
Comment (tmvumbi2) [inline configuration/concepts/adult_hiv_initial_form.csv]: @lauravignoli small typo  `Opportunitis` -> `Opportunistic`

File: configuration/concepts/adult_hiv_initial_form.csv [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+16c12c6b-f734-4fd6-974e-8088f5b1e013,,,La date de fin de traitement,End date of treatment,,,Misc,Date,,
+c1a0d403-2597-469c-a8df-8b424630f3ae,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Chimio prophylaxie INH,Chemo prophylaxis INH,,,Misc,Coded,,
+9d9a399a-d272-4d39-a5b3-e8f674aed8cf,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Diabétes?,Diabetes?,,,Misc,Coded,,
+6aa07a98-1ace-4cd5-bad6-7e04507bc0ab,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,HTA,HTA,,,Misc,Coded,,
+f6ddc619-4093-4077-b91a-ad54306243e1,,,B,B,,,Misc,N/A,,
+ceb11aff-09b7-4b1c-8ea8-7f98ef6079b1,,,C,C,,,Misc,N/A,,
+e765f0df-ab4d-47a8-a51d-6ca9277cae32,,,D,D,,,Misc,N/A,,
+56516172-9d35-4e52-9d98-00ff82235aa6,,f6ddc619-4093-4077-b91a-ad54306243e1;ceb11aff-09b7-4b1c-8ea8-7f98ef6079b1;e765f0df-ab4d-47a8-a51d-6ca9277cae32,Hépatites,Hepatitis(s),,,Misc,Coded,,
+83624a21-972b-40e2-af3e-1a42612f4756,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Autres maladies,Other diseases,,,Misc,Coded,,
+9200f505-f983-442b-99e7-23430e26e489,,,Drogue IV,Drogue IV,,,Misc,N/A,,
```
Comment (tmvumbi2) [inline configuration/concepts/adult_hiv_initial_form.csv]: @lauravignoli Was the English translation meant to be `Drug  IV`?

File: configuration/concepts/adult_hiv_initial_form.csv [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+a32c47d6-ff4b-4f0b-b8f6-dea611f17f59,,9200f505-f983-442b-99e7-23430e26e489,Toxicology,Toxicology,,,Misc,Coded,,
```
Comment (tmvumbi2) [inline configuration/concepts/adult_hiv_initial_form.csv]: @lauravignoli The French translation should read `Toxicologie`

File: configuration/concepts/adult_hiv_initial_form.csv [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+fd43f382-0f14-4740-ad8f-3ec38b05fd14,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,ATCD prescription ARV,ATCD prescription ARV,,,Misc,Coded,,
+02f7b1ff-1bfc-46b8-811d-bf524db681cb,,,Si traitement ARV antérieur; Date début,If previous ARV treatment; start date,,,Misc,Date,,
+fef74daa-2a47-4c4e-b9c2-d18078d8e3be,,,Date de fin (le cas échéant),End Date  ,,,Misc,Date,,
+81ff1578-da33-44f3-bdbe-71d7706ecd87,,,Formation sanitaire,Health Facility,,,Misc,Text,,
+09704fda-7fe3-45bf-a993-9b5eb7b69ed9,,,Transfert avec dossier,Transfer with file,,,Misc,N/A,,
+c74e2fd5-1005-49e8-bf2a-64aabd2c662a,,,Déjé été sou ARV,Already been ARV,,,Misc,N/A,,
```
Comment (tmvumbi2) [inline configuration/concepts/adult_hiv_initial_form.csv]: @lauravignoli Small typo on the french translation: `Déjé été sou ARV` -> `Déjà été sous ARV`

File: configuration/concepts/adult_hiv_initial_form.csv [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+51d7342d-d8ad-4061-97f3-c32724a47694,,,PTME,PMTCT,,,Misc,N/A,,
+aeecf2e8-4746-4023-bc39-efdd83ce4e94,,09704fda-7fe3-45bf-a993-9b5eb7b69ed9;c74e2fd5-1005-49e8-bf2a-64aabd2c662a;51d7342d-d8ad-4061-97f3-c32724a47694,Circonstances du traitement ARV antérieur,Circumstances of previous ARV treatment,,,Misc,Coded,,
+3b2a2ded-60f6-4818-a4f4-10f296f53ae1,,,Médicament ARV prescrits,Prescribed ARV drug,,,Misc,Text,,
+29538d59-7239-475d-b95c-3e493f8b88fe,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Traitement toujours en cours,Treatment still in progress,,,Misc,Coded,,
+db3088d9-cc5c-433f-8e47-5e29881dd9c5,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e;Do Not Know,Cotrimoxazole (Prophylaxie en cours),Cotrimoxazole (Prophylaxsis in progress),,,Misc,Coded,,
```
Comment (tmvumbi2) [inline configuration/concepts/adult_hiv_initial_form.csv:80]: @lauravignoli  Could you please double check the above concept? Noticed the following issues:
`Fully specified name:fr` =`Do Not Know` : the text is in English
`Fully specified name:en` = `Cotrimoxazole (Prophylaxie en cours)`: the text is in French and seems unrelated to its fr counterpart
`Short name:fr` =`Cotrimoxazole (Prophylaxsis in progress)`: the text is english
`Short name:en` is empty

File: configuration/concepts/adult_hiv_initial_form.csv [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+7d1187b3-f428-4064-80ec-869c16761793,23d94813-c123-4f6d-a7f8-66e22bc30e75;9a7ad1b1-942f-43b4-b280-5df81b515135;cf74d45d-7a99-4c12-9bbb-02095726f666;4c2629fe-5b75-4721-aec2-d5b792c7eb8e,,III. ANTECEDENTS DU PATIENT,III. PATIENT BACKGROUND,,,Misc,N/A,,
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,,Date de début de traitement ARV,ARV treatment start date,,,Misc,Date,,
+20b2eac4-ff3a-4569-b610-067c4cd8b027,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Traitements,Treatments,,,Misc,N/A,,
+059b8309-4cba-46cc-a857-e1e74be3fa86,20b2eac4-ff3a-4569-b610-067c4cd8b027,,IV. VISITE INITIALE,IV. INITIAL VISIT,,,Misc,N/A,,
+1fb2dd86-53b5-4815-9c64-edc081b908d9,83e4df0e-5723-4c53-ae8b-7411b2337bef;7d1187b3-f428-4064-80ec-869c16761793;059b8309-4cba-46cc-a857-e1e74be3fa86,,FICHE DU PATIENT AVEC VIH - ADULTE,PATIENT WITH HIV - ADULT INITIAL,,,Misc,N/A,,
+9d91cb77-3f10-11e4-adec-0800271c1b75,18351dd1-078e-456c-8c23-915b16e9dfaf;c36a7537-3f10-11e4-adec-0800271c1b75;1fb2dd86-53b5-4815-9c64-edc081b908d9,,All Observation Templates,All Observation Templates[FR],,,ConvSet,N/A,,
```
Comment (tmvumbi2) [inline configuration/concepts/adult_hiv_initial_form.csv]: @lauravignoli this change will be overridden in `/child_hiv_initial_form.csv`, I would like to suggest we remove it here

**Review by tmvumbi2 — CHANGES_REQUESTED** [review]

File: configuration/concepts/adult_hiv_initial_form.csv (hunk 1 above)
Comment (lauravignoli) [reply configuration/concepts/adult_hiv_initial_form.csv:12]: @tmvumbi2 This is working as expected to me? The possible answers are Positive, Negative or Do Not Know.

statusPartner

File: configuration/concepts/adult_hiv_initial_form.csv (hunk 7 above)
Comment (lauravignoli) [reply configuration/concepts/adult_hiv_initial_form.csv:80]: I decided to use some concepts with the name instead of uuid after I had to update the uuid because it was a duplicate. When there's no duplicates (which should be always) uuid or concept name can be used in-distinctively.

Remember that in this file, for the name, the first column is fr and the second en. I removed the short names when they are the same as the full name.

File: configuration/concepts/adult_hiv_initial_form.csv (hunk 1 above)
Comment (tmvumbi2) [reply configuration/concepts/adult_hiv_initial_form.csv:12]: @lauravignoli oh you right, I didn't see that this was an answer list

File: configuration/concepts/adult_hiv_initial_form.csv (hunk 7 above)
Comment (tmvumbi2) [reply configuration/concepts/adult_hiv_initial_form.csv:80]: Thanks for the clarification @lauravignoli, I missed that  'Do Not Know' was part of the answer

**Review by tmvumbi2 — APPROVED** [review]
